style.css

```diff
@@ -1 +1,302 @@
-/*  Add your styling here */
+ /* General */
+ * {
+     box-sizing: border-box;
+ }
+
+ html,
+ body {
+     margin: 0;
+     padding: 0;
+     font-family: 'Roboto', sans-serif;
+     line-height: 1.4;
+     text-align: center;
+     background: #F6C973;
+ }
+
+ .small-font {
+     font-size: 1rem;
+     vertical-align: middle;
+ }
+
+ /* Header */
+ /* [ON SMALL SCREENS] */
+ .page-nav {
+     width: 100%;
+     height: 3.5rem;
+     background: #E58143;
+     /* background: red; */
+     position: fixed;
+     top: 0;
```

Comment:
By default, if not defined, this value would already be set to zero so it wouldn't be necessary to define this.